webpack/scenes/ContentViews/Details/Filters/MatchContentModal/CVRpmMatchContentModal.js

```diff
@@ -17,9 +20,9 @@ const columnHeaders = [
   __('Summary'),
 ];
 
-const emptyContentTitle = __('No matching RPM found.');
-const emptyContentBody = __("Given criteria doesn't match any RPMs. Try changing your rule.");
-const emptySearchTitle = __('Your search returned no matching RPMs.');
+const emptyContentTitle = __('No matching non-modular package found.');
```

Comment:
```suggestion
const emptyContentTitle = __('No matching non-modular RPM found.');
```

We standardize on "RPM" over "Package" for the rest of the RPM filter page.